doc/CMakeLists.txt

```diff
@@ -2,18 +2,50 @@ file(GLOB json_files
   *.json
   )
 
-file(GLOB json_schema_files
-  *schema.json
-  )
-
 foreach(json_file ${json_files})
   get_filename_component(json_name ${json_file} NAME_WE)
   add_test(NAME jsonlint_${json_name}
     COMMAND jsonlint ${json_file})
 endforeach()
 
-foreach(json_schema_file ${json_schema_files})
-  get_filename_component(json_name ${json_schema_file} NAME_WE)
-  add_test(NAME ajv_${json_name}
-    COMMAND ajv compile -s ${json_schema_file})
-endforeach()
+set(BASELINE ${CMAKE_SOURCE_DIR}/doc/)
+set(COMMON_SCHEMA ${BASELINE}/common-schema.json)
+set(SR_COMMON_SCHEMA ${BASELINE}/sr-common-schema.json)
+
+set(AJV_OPTIONS --all-errors)
+
+get_filename_component(json_name common-schema.json NAME_WE)
```

Comment:
This is redundant. I used it to extract file name from the full path, which is no-op here.